packit/constants.py

```diff
@@ -190,6 +190,11 @@
     "```\ngit fetch https://src.fedoraproject.org/forks/{user}/rpms/{package}.git "
     "refs/heads/*:refs/remotes/{user}/*\n"
     "git checkout {user}/{branch}\n```\n"
+    "\n\n"
```

Comment:
I would probably separate it in a similar way as the instructions for additional changes:

```suggestion
    "\n---\n\n"
```